MySQL Datasource: Add timezone parameter (#27535)
grafana/grafana: docs/sources/features/datasources/mysql.md

*Database* | Name of your MySQL database.
*User* | Database user's login/username
*Password* | Database user's password

> *Session Timezone* | Specify the time zone used in the database session, e.g. `Europe/Berlin` or `+02:00`. The value will be set int the session with `SET time_zone='...'`. If you leave this field empty, the timezone will not be changed. You can find more information in the [MySQL documentation](https://dev.mysql.com/doc/refman/8.0/en/time-zone-support.html). 

Reviewer: ```suggestion
*Session Timezone* | Specify the time zone used in the database session, such as `Europe/Berlin` or `+02:00`. Set the value in the session with `SET time_zone='...'`. If you leave this field empty, then the time zone will not be changed. You can find more information in the [MySQL documentation](https://dev.mysql.com/doc/refman/8.0/en/time-zone-support.html). 
```

> *Session Timezone* | Specify the time zone used in the database session, such as `Europe/Berlin` or `+02:00`. Set the value in the session with `SET time_zone='...'`. If you leave this field empty, then the time zone will not be changed. You can find more information in the [MySQL documentation](https://dev.mysql.com/doc/refman/8.0/en/time-zone-support.html). 

Reviewer 2: ```suggestion
*Session Timezone* | Specify the time zone used in the database session, such as `Europe/Berlin` or `+02:00`. Set the value in the session with `SET time_zone='...'`. If you leave this field empty, then the time zone is not updated. For more information, refer to the [MySQL documentation](https://dev.mysql.com/doc/refman/8.0/en/time-zone-support.html). 
```

Author: Updated in merge commit

Author: Updated after merge
